data/data.json

```diff
@@ -10715,7 +10848,7 @@
  },
  "RKEDefaultK8sVersions": {
   "0.3": "v1.16.3-rancher1-1",
-  "default": "v1.22.7-rancher1-2"
+  "default": "v1.22.8-rancher1-1"
```

Comment:
@kinarashah I saw you put in a PR a while back to roll back the default version of kubernetes from 1.22 to 1.21 because 1.22 is experimental. Is this change correct?

Reply:
On Rancher 2.6.4 I see `1.23.4` is experimental and `1.22.7` is default.

Reply:
Kinara's PR: https://github.com/rancher/kontainer-driver-metadata/pull/880

Ryan, you are correct. That other PR is basically just switching KDM from rancher `release` back to `dev` branch where kubernetes 1.22 is listed as experimental. But it's open, so until it's merged this PR is correct. Waiting on discussion to confirm whether ok to merge or not

Reply:
Got input from @thedadams - _Just talked to Denise: she said we aren't going to merge Kinara's PR, so we are good to go with the RKE one._ Good to merge